fix(deps): Bump versions

Haxikowy opened the pull request.

Haxikowy committed:
chore: Bump deps

Haxikowy committed:
fix(ci): Bump node version to 20

ol-web commented:
hmm, a ta beta byla przetestowana chociazby na react native i nuxt 3? bo cos czuje ze moga byc zgrzyty przy tylu bumpach i jak to nie jest konieczne to moze zostac w becie

adrianklimek commented on .github/workflows/release.yml:
```diff
@@ -28,7 +28,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: 20
```
We should also update nvmrc

semantic-release-bot committed:
chore(release): 1.3.7-beta.1 [skip ci]

## [1.3.7-beta.1](https://github.com/SparingSoftware/eslint-config/compare/v1.3.6...v1.3.7-beta.1) (2024-04-16)

### Bug Fixes

* **ci:** Bump node version to 20 ([39ce34b](https://github.com/SparingSoftware/eslint-config/commit/39ce34bead44138223ba2e665a6261974039592b))
